### DISCUSSION TITLE
How to evaluate the model

### DISCUSSION THREAD

**Discussion by ChengQIAN93** [mir-group/nequip#453] [Q&A]
Dear developers, 

Recently, I try to used NequIP, and train a model successfully.

Now, I am trying to used ( nequip-evaluate --train-dir /path/to/training/session/ ) to evaluate the model, but failed, and shows the following message, and there is no output. 

![1721834120490](https://github.com/user-attachments/assets/5e979a0b-212a-40dc-a547-0c31a23bff52)

Could you let me know how to solve this problem ?

Thank you.

**Comment by ChengQIAN93** [comment]
aha, I solve the problem.

**Reply by mariummou** [reply]
what did you do? Thanks

**Reply by ChengQIAN93** [reply]
In the in the train.xyz, I have 1303 frames, at beginging, I set n_train: 1000   n_val: 303 in the input.yaml, so there is no frames for testing. Then I changed  n_val: 303 to  n_val: 200, it works.
